demo-project/src/main/resources/flyway/migrations/V2__add_id_column.sql

```diff
@@ -0,0 +1,3 @@
+alter table public.flyway_test drop constraint flyway_test_pkey;
+ALTER TABLE public.flyway_test
+ADD COLUMN ID SERIAL PRIMARY KEY;
```

Comment:
We can't make this the primary key in a migration; existing rows won't have a value here.